base/sysimg.jl

```diff
@@ -28,6 +28,7 @@ let
             :Serialization,
             :Libdl,
             :Markdown,
+            :MbedTLS,
```

Comment:
@fredrikekre, if I just remove this, will MbedTLS still be accessible like `using MbedTLS`, but it won't be precompiled? Does anyone think that's worth it? (i.e. smaller sysimg size since MbedTLS module doesn't really need to be precompiled)

Reply:
Correct (I think). One advantage is that you can still load a (different) package named MbedTLS :)

Reply:
Yes, that seems to work as expected. I'll remove it from this list then.